src/power_udev/provider_power_udev.cpp

```diff
@@ -239,17 +277,19 @@ static char const * get_chg_type_name(ChargerType t)
     return names[cor::enum_index(t)];
 }
 
+static const std::map<std::string, ChargerType> charger_types = {
+    {"USB_DCP", ChargerType::DCP},
+    {"USB", ChargerType::USB},
+    {"Mains", ChargerType::Mains},
+    {"CDP", ChargerType::CDP},
+    {"USB_CDP", ChargerType::DCP},
+    {"USB_ACA", ChargerType::USB}
+};
+
 static ChargerType charger_type(std::string const &v)
 {
-    return (v == "USB_DCP"
-            ? ChargerType::DCP
-            : (v == "Mains"
-               ? ChargerType::Mains
-               : (v == "USB"
-                  ? ChargerType::USB
-                  : (v == "DCP"
-                     ? ChargerType::CDP
```

Comment:
The old code here maps `"DCP"` to `ChargerType::CDP`, that's missing in the mapping above? Or was that a bug in the old code?